docs/examples.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[examples]]
+== Examples
+
+This page contains the information you need to perform various {es} operations 
+by using the Client.
+
+
+[discrete]
+[[indexing-net]]
+=== Indexing a document
+
+Indexing a document is as simple as:
+
+[source,csharp]
+----
+var tweet = new Tweet
+{
+    Id = 1,
+    User = "kimchy",
+    PostDate = new DateTime(2009, 11, 15),
+    Message = "Trying out NEST, so far so good?"
+};
+
+var response = client.Index(tweet, idx => idx.Index("mytweetindex")); //or specify index via settings.DefaultIndex("mytweetindex");
```

Comment:
I'd like to drop an sync examples if possible. It's generally bad and for most modern situations not recommended. Could we default to the async example and maybe just mention once, the sync alternative if there's no alternative?